src/components/Project/ImageList.vue

```diff
@@ -22,12 +22,11 @@ const emit = defineEmits(['selectImage'])
 
 const headers = ref([
   { title: 'IMAGE', align: 'start', sortable: false, key: 'url' },
-  { title: 'FILTER', align: 'start', sortable: true, key: 'FILTER' },
-  { title: 'IMAGE NAME', align: 'start', sortable: true, key: 'basename' },
   { title: 'TARGET', align: 'start', sortable: true, key: 'target_name' },
+  { title: 'FILTER', align: 'start', sortable: true, key: 'FILTER' },
```

Comment:
use `primary_optical_element` instead of `FILTER` for getting the filter from the /frames query